### ISSUE TITLE
Ogg not working!

### ISSUE THREAD

**Issue by Willymcmilly** [CapsAdmin/pac3#111]
I know people have been complaining about this before but I need an answer, i'm using puush and i'm using a direct link to an .ogg file from a public folder.
http://puu.sh/eD1XT.ogg
The problem is that it says
[PAC3] OGG error: stream[0x436e1708][7][http://puu.sh/eD1XT.ogg] reason: download failed
Puush links work in 3D Streaming Radio by what i've learned.
I've also tried dropbox(which sucked) and copy.com(which I had the exact same problem with)
Please give me an answer, based CapsAdmin!

**Comment by LUModder** [comment]
Go back to using dropbox and follow this.

http://steamcommunity.com/groups/metastruct/discussions/0/34095132064688951/

**Comment by Ameno-Sagiri** [comment]
Dropbox and Copy work for me. I used to use OGGs and webaudio all the time, but I realized how unreliable both of them are. Webaudio is way too quiet, but it usually always loads up properly, and OGGs only seemed to work 10% of the time for me when I used them for 2 years. Most of the time they wouldn't download or initialize, but sometimes they worked. 

Another problem with audio in PACs, is that if everyone you're showing it to doesn't have the FOV Override enabled, every time they look slightly away from you, then back at you, the sound file will stop playing for them and won't reactivate until you toggle it again. They have to look at you the entire time you're playing the audio file.
